__tests__/flavored-compilers.test.js

```diff
@@ -33,12 +33,41 @@ describe('ReadMe Flavored Blocks', () => {
     `);
   });
 
+  it('Glossary Items', () => {
+    expect(compile(parse('<<glossary:owl>>'))).toMatchInlineSnapshot(`
+      "<<glossary:owl>>
+      "
```

Comment:
Should these have new lines like this?

Reply:
IIUC, `RDMD.parse` will wrap the inlines (glossary item or plain text) in a paragraph.

It could be that `RDMD.md` is just inserting an extra new line, but in either case, the resulting markdown is equivalent.